WordPressAuthenticator/Signin/LoginEmailViewController.swift

```diff
@@ -511,7 +519,9 @@ open class LoginEmailViewController: LoginViewController, NUXKeyboardResponder {
 
 // MARK: - Google Sign In
 
-// LoginFacadeDelegate methods for Google Google Sign In
+/// Make Google Sign In conform to the LoginFacade protocol.
+/// The delegate calls these methods from LoginFacade.m.
+///
```

Comment:
This is documentation I forgot to update for the previous segue removal, `.show2fa`